packages/shared/src/components/ShareBar.tsx

```diff
@@ -42,53 +53,155 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
     onClick(ShareProvider.CopyLink);
   };
 
+  const onShareToSquad = (squad: Squad) => {
+    trackEvent(postAnalyticsEvent(AnalyticsEvent.StartShareToSquad, post));
+    openModal({
+      type: LazyModal.CreateSharedPost,
+      props: {
+        squad,
+        preview: post,
+        onSharedSuccessfully: () => {
+          return trackEvent(
+            postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post),
+          );
+        },
+      },
+    });
+  };
+
   return (
     <WidgetContainer className="hidden laptop:flex flex-col p-3">
-      <p className="typo-callout text-theme-label-tertiary">
+      <p className="mb-4 typo-callout text-theme-label-tertiary">
         Would you recommend this post?
       </p>
-      <div className="inline-flex relative flex-row items-center mt-3">
+
+      <div className="flex flex-wrap gap-2">
         <SimpleTooltip content="Copy link">
-          <ShareButton
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             onClick={trackAndCopyLink}
             pressed={copying}
-            icon={<CopyIcon />}
-            className="btn-tertiary-avocado"
-          />
+            textPosition="justify-between"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <CopyIcon secondary className="w-7 h-7" />
+              </div>
+            }
+          >
+            Copy link
+          </Button>
         </SimpleTooltip>
+
         <SimpleTooltip content="Share on WhatsApp">
-          <ColorfulShareButton
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
```

Comment:
```suggestion
            className="group flex-col mb-2 font-normal text-center hover:bg-transparent border-0 share btn-tertiary typo-caption1"
```

The salt color is already the color of a tertiary button.
Also we should avoid using direct color names and prefer our named versions.